web/models/user.ex

```diff
@@ -7,6 +7,13 @@ defmodule CodeCorps.User do
     field :password, :string, virtual: true
     field :encrypted_password, :string
 
+    field :first_name, :string
+    field :last_name, :string
+
+    field :twitter, :string
+    field :biography, :string
+    field :website, :string
+
```

Comment:
If you're going for consistency I think the simplest consistency works best, otherwise there's always the question of when we reach too many, and then having to do a large refactoring to meet those constraints.

I find if I'm reading/scanning code, then alpha sub-ordering almost always makes it easier.